run.py

```diff
@@ -1044,6 +1044,36 @@ def supported_devices(
     def engine_manifest():
         return engine_manifest_loader.load_manifest()
 
+    @app.post(
+        "/validate_kana",
+        response_model=bool,
+        tags=["その他"],
+        summary="テキストがAquesTalk記法に従っているか判定する",
+        responses={
+            400: {
+                "description": "テキストが不正です",
+                "model": ParseKanaBadRequest,
+            }
+        },
+    )
+    def validate_kana(text: str):
+        """
+        テキストがAquesTalk記法に従っているかどうかを判定します.
+
+        Parameters
+        ----------
+        text: str
+            validateする対象の文字列
```

Comment:
特に規則は定めていないのですが、英単語をなるべく含めないドキュメントで揃えるのとかどうでしょう。
```suggestion
            判定する対象の文字列
```